pkg/network/setup/netstat_test.go

```diff
@@ -326,7 +326,7 @@ var _ = Describe("netstat", func() {
 		setup.NetStat.UpdateStatus(setup.Vmi, setup.Domain)
 
 		Expect(setup.Vmi.Status.Interfaces).To(Equal([]v1.VirtualMachineInstanceNetworkInterface{
-			newVMIStatusIface(networkName, nil, ifaceMAC, guestIfaceName, netvmispec.InfoSourceGuestAgent),
+			newVMIStatusIface(networkName, nil, ifaceMAC, guestIfaceName, netvmispec.InfoSourceDomainAndGA),
```

Comment:
Please add a test for sriov net device reported by the GA only.

Reply:
That is not possible.
If we see a rogue interface that does not appear in the domain, we cannot detect if it is SR-IOV or not.
There are tests for such interfaces, in the general sense.